lib/ansible/modules/user.py

```diff
@@ -2361,7 +2362,7 @@ def _change_user_password(self):
         if self.password:
             cmd += ['-passwd', '/Users/%s' % self.name, self.password]
         else:
-            cmd += ['-create', '/Users/%s' % self.name, 'Password', '*']
+            cmd += ['-passwd', '/Users/%s' % self.name, '']
```

Comment:
It seems like this would also change the behavior when password is None. Could the if statement be modified instead? `if self.password is not None:`

Reply:
This is exactly the expected behavior. Please see the reason in the "Additional information" section for why it is this way.